gfx/UltimateCataclysm/tile_config.json

```diff
@@ -258,7 +258,6 @@
             "mag_smg",
             "mag_archery",
             "mag_cooking",
-            "mag_fieldrepair",
```

Comment:
tile.config.json files are generated from files in tilesets repository, you want to remove it there instead:
https://github.com/I-am-Erk/CDDA-Tilesets/blob/6c67be12839e364498099fae945e661323b7f69e/gfx/UltimateCataclysm/pngs_small_20x20/items/books/texts.json#L106